### PR TITLE
toc, op-guide: spit configuration by component

### PR DESCRIPTION
**Pull request by dcalvin**
In line with: [TOC, op-guide: spit configuration by component #995](https://github.com/pingcap/docs/pull/995).
Need to wait for Morgan to respond to the missed config file in the TOC.

### CODE REVIEW

File: op-guide/pd-configuration.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+---
+title: PD 配置参数
+category: reference
+---
+
+# PD 配置参数
+
```
Comment (lilin90) [inline op-guide/pd-configuration.md]: Please also add a brief intro sentence like that in the English version.

File: op-guide/tidb-configuration.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
+---
+title: TiDB 配置参数
+category: reference
+---
+
+# TiDB 配置参数
+
+## 参数概述
```
Comment (lilin90) [inline op-guide/tidb-configuration.md]: Please remove this line.

File: op-guide/tikv-configuration.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+---
+title: TiKV 配置参数
+category: reference
+---
+
+# TiKV 配置参数
+
+TiKV 的命令行参数支持一些可读性好的单位转换。
+
++ 文件大小（以 bytes 为单位）: KB, MB, GB, TB, PB（也可以全小写）
```
Comment (lilin90) [inline op-guide/tikv-configuration.md]:
```suggestion
+ 文件大小（以 byte 为单位）：KB, MB, GB, TB, PB（也可以全小写）
```

File: op-guide/tikv-configuration.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
++ 时间（以毫秒为单位）: ms, s, m, h
```
Comment (lilin90) [inline op-guide/tikv-configuration.md]:
```suggestion
+ 时间（以毫秒为单位）：ms, s, m, h
```

**Comment by dcalvin** [comment]
@lilin90 Thanks. PTAL again.

File: op-guide/tidb-configuration.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
+---
```
Comment (lilin90) [inline op-guide/tidb-configuration.md]: Let's change the file name to `configuration.md` temporarily.

**Pull request merged**